server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -295,6 +300,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 		private readonly checkpointService: ICheckpointService,
 		private readonly restartOnCheckpointFailure: boolean,
 		private readonly kafkaCheckpointOnReprocessingOp: boolean,
+		private readonly deliCheckpointMetricInterval: number,
```

Comment:
Can you add this to the `IDeliServerConfiguration` object instead of an argument in the constructor?

Reply:
Moved in the configuration.